src/main/java/com/commercetools/pingpong/model/Game.java

```diff
@@ -54,38 +54,41 @@ public void updateScoreOfPlayer(Player player, int actionType) {
     }
 
     public void updateMatchScoreOfPlayers() {
+        if (hasPlayerWonSet(playerOne, playerTwo) && !isMatchOver()) {
+            this.playerOne.addMatchScore();
+            this.playerOne.reset();
+            this.playerTwo.reset();
 
+        } else if (hasPlayerWonSet(playerTwo, playerOne) && !isMatchOver()) {
+            this.playerTwo.addMatchScore();
+            this.playerOne.reset();
+            this.playerTwo.reset();
```

Comment:
It seems that these two lines of code: 
````java
this.playerOne.reset();
this.playerTwo.reset();
````
are used several times, maybe have a method here in the `Game` class:
````java
private void resetScores(){
    playerOne.reset();
    playerTwo.reset();
}
````

Then you can reuse the one-liner call `resetScores()` every time instead of two lines ;)